Gemfile

```diff
@@ -32,6 +32,8 @@ end
 
 group :test do
   gem "capybara", "~> 3.0"
+  # TEMP: list hashdiff explicitly in Gemfile to force 1.0.0.beta1; remove once 1.0.0 is released
```

Comment:
Thank you for this comment 🙇

Reply:
I am not watching the hashdiff project releases so hopefully I'll remember why (I have this same change on my portfolio) and update the Gemfiles